### PR TITLE
Fix missing terminating newline for TOML files

### LINKED ISSUE #152: Lost ability to properly parse TOML files that do not end with a newline character
In these cases we get a `Ez_toml.Types.Error (_, 0, 0)` exception.

### PR DESCRIPTION
**Pull request by lefessan**
fix #152

### CODE REVIEW

File: src/vendor/ez_toml/v1.ml [hunk 1]
```diff
@@ -38,7 +38,17 @@ module TOML = struct
           lexbuf.lex_curr_p <- {lexbuf.lex_curr_p with pos_fname = file}
     end;
     try
-      let lines = Internal_parser.toml Internal_lexer.tomlex lexbuf in
+      let eof = ref false in
```
Comment (nberth) [inline src/vendor/ez_toml/v1.ml:41]: I though the problem was a missing newline expected by the parser. Is this actually fixing the issue?

Comment (lefessan) [reply src/vendor/ez_toml/v1.ml:41]: The idea is to always send a EOL token before the EOF token, so there is always an EOL at the end...

Comment (lefessan) [reply src/vendor/ez_toml/v1.ml:41]: And yes, I added a test in the original `ez_toml` and the problem is fixed there.

Comment (nberth) [reply src/vendor/ez_toml/v1.ml:41]: Ah sure! I originally did not distinguish `EOF` and `EOL`.

**Review by nberth — APPROVED** [review]

**Pull request merged**